desktop-app/app/components/ErrorBoundary/index.js

```diff
@@ -1,4 +1,27 @@
 import React, {Fragment} from 'react';
+import TextAreaWithCopyButton from '../../utils/TextAreaWithCopyButton';
+import Button from '@material-ui/core/Button';
+
+const {openNewGitHubIssue} = require('electron-util');
+
+const reportBody = `Hi I'm reporting an app crash:
+<details>
+  <summary>Environment</summary>
+</details>
+<details>
+  <summary>Error Info</summary>
+</details>
+<details>
+  <summary>Stack Trace</summary>
+</details>`;
+
+const createIssue = () => {
+  openNewGitHubIssue({
+    user: 'responsively-org',
+    repo: 'responsively-app',
+    body: reportBody,
```

Comment:
@gorogoroumaru could you please add also a `title` to the issue pre-filled info, something like `App crash report`
it seems that `openNewGitHubIssue` supports it since it uses [new-github-issue-url](https://github.com/sindresorhus/new-github-issue-url#title)